scripts/build.ps1

```diff
@@ -105,7 +105,7 @@ try {
       }
     }
 
-    & $VcpkgExe install freetype glad glm lua sdl2 sdl2-image sdl2-mixer[core,mpg123,libvorbis] --triplet x64-windows
+    & $VcpkgExe install freetype glad glm lua sdl2 sdl2-image sdl2-mixer[core,mpg123,libvorbis] spdlog --triplet x64-windows
```

Comment:
Why is this change in here? Is this some bugfix?

Reply:
This is so that the automated builds properly build with spdlog, otherwise it would not be pulled in and compilation would fail.